src/js/actions/pluginSettings.js

```diff
@@ -68,11 +68,19 @@ export function asyncPluginFetchSettings(zoneId) {
 export function asyncPluginUpdateSetting(settingName, zoneId, value) {
     return (dispatch, getState) => {
         let oldSetting = getState().pluginSettings.entities[zoneId][settingName];
+
+        if (settingName == "default_settings") {
```

Comment:
The `_SUCCESS` and `_ERROR` options update `isFetching`.  You can tell whether the call has completed based on that.